lib/vector-core/src/event/metric/mod.rs

```diff
@@ -476,6 +479,15 @@ impl Finalizable for Metric {
     }
 }
 
+impl GetEventCountTags for Metric {
+    fn get_tags(&self) -> EventCountTags {
+        let source = self.metadata().source_id().map(|output| output.to_string());
+        // Metrics do not contain a service field.
```

Comment:
Is this an intentional choice? I don't see a real-life reason why metrics couldn't be associated with a service.

Reply:
Currently only Log events can have schemas. Once we expand it so that a meaning can point to a metric tag then we will be able to extract the service here.

Reply:
Just want to double check that we will be emitting the `service` tag for logs, metrics, and traces. If we need to follow up on metrics and traces we can do so, but I think we do need to do that before calling this project "finished".

Reply:
Ah, I didn't consider the `service` tag to be part of the schema - just thought we'd add a metric tag of `service: foo`